perses/annihilation/ncmc_switching.py

```diff
@@ -135,6 +136,29 @@ def _getAvailableParameters(self, system, prefix='lambda'):
                         parameters.append(parameter_name)
         return parameters
 
+    def _updateAlchemicalState(self, context, functions, value):
+        """
+        Update alchemical state using the specified lambda value.
+
+        Parameters
+        ----------
+        context : simtk.openmm.Context
+            The Context
+        functions : dict
+            A dictionary of functions
+        value : float
+            The alchemical lambda value
+
+        TODO: Improve function evaluation to better match Lepton and be more flexible in exact replacement of 'lambda' tokens
+
+        """
+        from parsing import NumericStringParser
```

Comment:
Actually, @juliebehr, I think I fucked up the imports if you're using python 3. Sorry about that!  Fixed now, I hope.